### PR TITLE
Admin ui fixes

### PR DESCRIPTION
**Pull request by MadeByMike**
Fixes a few minor UX issues in the Admin UI block editor.

### CODE REVIEW

File: website/field-content/src/views/editor/add-block.js [hunk 1]
```diff
@@ -162,6 +162,7 @@ let AddBlock = ({ editorState, editor, blocks }) => {
 							padding: 0,
 							margin: 0,
 							border: 'solid 1px #eaeaea',
+							zIndex: '9999999999999',
```
Comment (dominikwilkowski) [inline website/field-content/src/views/editor/add-block.js]: 🤦‍♂

File: website/src/components/a11y-filter.js [hunk 2]
```diff
@@ -4,7 +4,12 @@ import { jsx, useBrand } from '@westpac/core';
 import Select from 'react-select';
 
 const SVGFilters = () => (
-	<svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" version="1.1">
+	<svg
+		aria-hidden="true"
+		style={{ display: 'none' }}
```
Comment (dominikwilkowski) [inline website/src/components/a11y-filter.js]: did you cross browser check this? I think adding `display:none` to svgs breaks safari or firefox?

File: website/field-content/src/views/editor/add-block.js (hunk 1 above)
Comment (MadeByMike) [reply website/field-content/src/views/editor/add-block.js]: Sorry not sorry

File: website/src/components/a11y-filter.js (hunk 2 above)
Comment (MadeByMike) [reply website/src/components/a11y-filter.js]: I did not /honesty

Comment (MadeByMike) [reply website/src/components/a11y-filter.js]: Dom it pains me to say it but you are correct!

Comment (MadeByMike) [reply website/src/components/a11y-filter.js]: Fix on the way

File: website/field-content/src/views/editor/add-block.js (hunk 1 above)
Comment (MadeByMike) [reply website/field-content/src/views/editor/add-block.js]: There is a bit of "whack a mole" with z-indexes at the moment. We should revisit this later

**Pull request merged**